server/utils/scripts.js

```diff
@@ -0,0 +1,135 @@
+const { spawn } = require('node:child_process')
+
+const absoluteBasePath = "/Users/adam/assetto-server-test-files"
```

Comment:
Make this an environment variable and make sure that a check is made to verify if the path is valid